examples/incito-publication-template.html

```diff
@@ -68,5 +68,13 @@
             data-translation-keys--publication-viewer-download-button="Download Now"
             defer
         ></script>
+
+        <script>
+            document
+                .querySelector('#publication-container')
+                .addEventListener('tjek-publication:rendered', (e) => {
```

Comment:
Lose the tjek- because we're using the container element here. Plus, we don't use the word Tjek in the SDK right now (SGN).